docs/unified-test-documentation/dasharo-compatibility/346-hotplug.md

```diff
@@ -0,0 +1,138 @@
+# Dasharo compatibility: Hot-plug
+
+## Test cases common documentation
+
+**Test setup**
+
+1. Proceed with the
+   [Generic test setup: firmware](../../generic-test-setup/#firmware).
+1. Proceed with the
+   [Generic test setup: OS installer](../../generic-test-setup/#os-installer).
+1. Proceed with the
+   [Generic test setup: OS installation](../../generic-test-setup/#os-installation).
+1. Proceed with the
+   [Generic test setup: OS boot from disk](../../generic-test-setup/#os-boot-from-disk).
+
+## HOT001.001 SATA hot-plug detection (firmware)
+
+**Test description**
+
+This test aims to verify that the disk connected to the SATA port could be
+detected after hot-plug.
+
+**Test configuration data**
+
+1. `FIRMWARE` = Dasharo
+
+**Test setup**
+
+1. Proceed with the
+   [Test cases common documentation](#test-cases-common-documentation) section.
+
+**Test steps**
+
+1. Power on the DUT.
+1. Hold the `BOOT_MENU_KEY` to enter the UEFI Boot Menu.
+1. Connect the disk to the SATA port.
+1. Press `ESC` to go to Setup Menu.
+1. Select the `One Time Boot` position using arrows and click `Enter`.
```

Comment:
And what if the SATA disk is empty (with no OS)?

Reply:
Added step in test setup